trio/_highlevel_open_tcp_listeners.py

```diff
@@ -138,7 +139,7 @@ async def open_tcp_listeners(port, *, host=None, backlog=None):
             errno.EAFNOSUPPORT,
             "This system doesn't support any of the kinds of "
             "socket that that address could use",
-        ) from trio.MultiError(unsupported_address_families)
+        ) from MultiError(unsupported_address_families)
```

Comment:
I think you may as well just change this to ExceptionGroup, since as a `__cause__` it's not directly exposed to users' `except` clauses

Reply:
Done.